pds_doi_service/api/controllers/dois_controller.py

```diff
@@ -114,13 +114,21 @@ def _records_from_dois(dois, node=None, submitter=None, osti_label=None):
     """
     records = []
 
+    list_action = DOICoreActionList(db_name=_get_db_name())
+
     for doi in dois:
+        # Pull info from transaction database so we can get the most accurate
+        # info for the DOI
+        list_kwargs = {'lidvid': doi.related_identifier}
+        list_result = json.loads(list_action.run(**list_kwargs))[0]
```

Comment:
Maybe this is the accepted idiom with `list_action.run` instead of the simpler `list_action.run(lidvid=doi.related_identifier)` but hey I just work here 😅